test/util/core/iterable-to-string-test.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var referee = require("referee");
+var iterableToString = require("../../../lib/sinon/util/core/iterable-to-string");
+var assert = referee.assert;
+
+describe("util/core/iterable-to-string", function () {
+    it("returns an String representation of Array objects", function () {
+        var arr = [1, "one", true, undefined, null];
+        var expected = "1,'one',true,undefined,null";
+
+        assert.equals(iterableToString(arr), expected);
+    });
+
+    it("returns an String representation of Map objects", function () {
+        if (typeof Map === "function") {
```

Comment:
Wouldn't it be better to wrap the `it` statement, like you did in `match-test.js`?